lisa/tools/hyperv.py

```diff
@@ -445,6 +445,76 @@ def enable_internal_dhcp(self, dhcp_scope_name: str = "DHCPInternalNAT") -> None
         # Restart the DHCP server to apply the changes
         service.restart_service("dhcpserver")
 
+    # The default lisa working path is under 'C:/' which is not ideal for large data.
+    # This method initializes an attached disk, creates a partition, formats it,
+    # and mounts it to the lisa working directory
+    # This method expects an un-initialized (RAW) disk to be available on Hyper-V host.
+    def use_raw_disk_for_lisa_working_dir(self) -> None:
+        powershell = self.node.tools[PowerShell]
+
+        # Get the first disk that is not initialized
+        raw_disks = powershell.run_cmdlet(
+            "Get-Disk | Where-Object PartitionStyle -Eq 'RAW'",
+            force_run=True,
+            output_json=True,
+        )
+        # If no un-initialized disk is found, log a warning and return
+        if not raw_disks:
+            self._log.warning("No un-initialized disk found.")
```

Comment:
1. Don't print warning, either print info or raise exception.
2. The following log line should be merged into one line, so people always know they are logged together.